targets/ingester/.eslintrc.json

```diff
@@ -1,8 +1,17 @@
 {
-  "extends": ["@socialgouv/eslint-config-recommended"],
+  "extends": [
+    "@socialgouv/eslint-config-recommended",
+    "plugin:import/typescript"
+  ],
   "settings": {
     "jest": {
       "version": 26
     }
-  }
+  },
+  "overrides": [
+    {"files": ["**/*.ts", "**/*.tsx"], "extends":"plugin:@typescript-eslint/recommended","rules": {
```

Comment:
Might want to replace it by [`@socialgouv/eslint-config-typescript`](https://github.com/SocialGouv/linters/blob/master/packages/eslint-config-typescript/README.md)